VortexEngine/src/patterns/multi/DripMorphPattern.h

```diff
@@ -0,0 +1,41 @@
+#ifndef DRIPMORPH_PATTERN_H
+#define DRIPMORPH_PATTERN_H
+
+#include "MultiLedPattern.h"\
+
+#include "../../Timer.h"
+
+class DripMorphPattern : public MultiLedPattern
+{
+public:
+  DripMorphPattern();
```

Comment:
There is m_blinkOnDuration and m_blinkOffDuration members but no way to initialize them in the constructor

Reply:
same with m_speed, maybe should be exposed through parameter as well

Reply:
yeah that's what was supposed to happen but I missed it woops